doc/source/whatsnew/v0.23.0.txt

```diff
@@ -491,7 +491,7 @@ Groupby/Resample/Rolling
 Sparse
 ^^^^^^
 
--
+- Bug in :class:`SparseArray` where if a scalar and index are passed in it will coerce to float64 regardless of scalar's dtype. (:issue:`19163`)
```

Comment:
Could you clarify what "passed in" refers to here? Is it specifically `.assign`? Or any method setting / updating the sparse array?